var/spack/repos/builtin/packages/py-salib/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PySalib(PythonPackage):
+    """Python implementations of commonly used sensitivity analysis methods."""
+
+    homepage = "http://salib.readthedocs.org"
+    git      = "https://github.com/SALib/SALib"
```

Comment:
Can we download from PyPI instead? GitHub tags are moving targets, so they can't be checksummed.

Reply:
That's what I tried first (and the pip and wheel dependencies were leftovers from that), but it failed with:

```
  File "/opt/spack/opt/spack/linux-ubuntu20.04-haswell/gcc-9.3.0/py-setuptools-scm-4.1.2-chkgdqej5d67pbvg377qdxzdzuhkee6k/lib/python3.8/site-packages/setuptools_scm/hacks.py", line 28, in fallback_version
    if config.parentdir_prefix_version is not None:
AttributeError: 'Configuration' object has no attribute 'parentdir_prefix_version'
```